src/test/db.rs

```diff
@@ -362,3 +369,22 @@ async fn index_option_defaults_test(defaults: Option<IndexOptionDefaults>, name:
     };
     assert_eq!(event_defaults, defaults);
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn watch_test() {
+    let _guard = LOCK.run_concurrently().await;
+
+    let client = EventClient::new().await;
+    if client.server_version_lt(4, 0) || client.is_standalone() {
+        return;
+    }
+    let db = client.database("test_db");
```

Comment:
ditto here use the function name for this.

Reply:
fixed

Reply:
I'm now realizing that this test has the same name as the other watch tests, so there will be some collisions. Can you rename each of the tests to be unique?

Reply:
makes sense, fixed